etlhelper/db_helpers/informix.py

```diff
@@ -0,0 +1,79 @@
+"""
+Database helper for Postgres
+"""
+from contextlib import contextmanager
+
+from etlhelper.db_helpers.db_helper import DbHelper
+
+
+class InformixDbHelper(DbHelper):
+    """
+    Postgres db helper class
+    """
+    def __init__(self):
+        super().__init__()
+        try:
+            import ibm_db_dbi
+            self.sql_exceptions = (ibm_db_dbi.ProgrammingError)
+            self._connect_func = ibm_db_dbi.connect
+            self.connect_exceptions = (ibm_db_dbi.OperationalError)
+            self.required_params = {'hostname', 'port', 'database', 'uid'}
+        except ImportError:
+            print("The Informix Python libraries could not be found.\n"
+                  "Run: python -m pip install ibm_db")
+
+    def get_connection_string(self, db_params, password_variable):
+        """
+        Return a connection string
+        :param db_params: DbParams
+        :param password: str, password
+        :return: str
+        """
+        # Prepare connection string
+        password = self.get_password(password_variable)
+        return f'database={db_params.database};hostname={db_params.hostname};' \
+               f'port={db_params.port};protocol=tcpip;uid={db_params.uid};pwd={password}'
+
+    def get_sqlalchemy_connection_string(self, db_params, password_variable):
+        """
+        Returns connection string for sql alchemy
+        """
+        password = self.get_password(password_variable)
```

Comment:
Refactor for Informix connection string.  This is prime TDD territory once you have manually figured out what the SQLAlchemy string is supposed to look like.